app/controllers/users_controller.rb

```diff
@@ -3,6 +3,7 @@ class UsersController < ApplicationController
 
   def edit
     @user = current_user
+    @active_casa_admins = CasaAdmin.where(casa_org_id: @user.casa_org_id).active
```

Comment:
Alternatively, there's a scope for `in_organization` which could be combined with the `current_organization` helper.

```ruby
@active_casa_admins = CasaAdmin.in_organization(current_organization).active
```

Reply:
Good call, thank you!!